build.sh

```diff
@@ -2,12 +2,13 @@
 
 set -e
 
+# configuration
 KERNEL_VERSION_RPI1=4.4.0-1-rpi
 KERNEL_VERSION_RPI2=4.4.0-1-rpi2
 
-INSTALL_MODULES="kernel/fs/btrfs/btrfs.ko"
-INSTALL_MODULES="$INSTALL_MODULES kernel/drivers/scsi/sg.ko"
-INSTALL_MODULES="$INSTALL_MODULES kernel/net/ipv6/ipv6.ko"
+INSTALL_MODULES=("kernel/fs/btrfs/btrfs.ko")
```

Comment:
See https://github.com/debian-pi/raspbian-ua-netinst/pull/236
Given the comments in that PR, we apparently already require bash 4. Although I don't think it's unreasonable, being fully compatible with 3.1+ would be even better (which this PR seems to be)